src/rio.js

```diff
@@ -119,13 +119,13 @@ export class Rio {
   }
 
   /**
-   * Set option to append unused query params for endpoint.
-   * Enabling this option, all params that were not used by
+   * Set option to ingore unused query params for endpoint.
+   * Disabling this option, all params that were not used by
    * an endpoint template will be resolved into query params
    * as 'key=value' pairs and concatenated to the endpoint.
    */
-  enableAppendOfUnusedQueryParams() {
-    this.appendUnusedQueryParams = true;
+  setIgnoreUnusedActionParams(ignoreUnusedActionParams) {
```

Comment:
give it a default value? So that it doesn't end up `undefined`. I know the logic will still be okay, but it's more explicit if we hold on to just true/false values.